### PR TITLE
Ste post match

### PR DESCRIPTION
**Pull request by ilPhil**

### CODE REVIEW

**Review by arol — COMMENTED** [review]
You don't need to put your name in the PR names, just be descriptive on what the code adds/fix to the existing product.

File: controllers/match.controller.js [hunk 1]
```diff
@@ -65,12 +66,48 @@ exports.getMatch = async (ctx) => {
...
 };
 
-
 exports.createMatch = async (ctx) => {
   try {
-    //ctx.body = await matchModel._update(ctx.params.matchId, (ctx.params.action+'ed').toUpperCase());
-    ctx.body = ctx.request.body
-    console.log('esisto', ctx.request.body)
+
+    const leagues = await leagueModel._getOne(ctx.request.body.league_id);
```
Comment (arol) [inline controllers/match.controller.js]: Calling an encapsulated or "private" method of another component.

a.k.a. remove that underscore from `_getOne`.

File: models/match.model.js [hunk 2]
```diff
@@ -53,6 +53,16 @@ exports.getOne = (matchId) => {
   });
 }
 
+exports.post = (ul1,ul2,ctx) => {
+  return new Promise ((resolve, reject) => {
```
Comment (arol) [inline models/match.model.js:57]: Try to refactor this in a new task/issue. Create the promise version of the query method on the `conn` object once (`db.js`?) so you don't have to do it in all your models and methods.

**Review by arol — CHANGES_REQUESTED** [review]
Just remove that underscore from `_getOne`. The other comment is just suggestions for the future.

**Comment by ilPhil** [comment]
Thanks Arol, I try to solve immediately

**Pull request merged**